conftest.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from Library.web_utility import GenericMethod
+from Library.file_library import ReadJson
+from config import *
+from selenium import webdriver
+
+""" This is the conftest file which contains commonly used file in all pages """
+
+options = webdriver.ChromeOptions()
+options.add_argument("--disable-notifications")
+driver = webdriver.Chrome(executable_path, options=options)
```

Comment:
You can have these code in a fixture and make sure fixture should give you the driver as output.

Reply:
I have used it inside fixture and used request object to over come it